### DISCUSSION TITLE
Current wallet application

### DISCUSSION THREAD

**Discussion by Gazarta** [IBAX-io/go-ibax#207] [Q&A] [answered]
i think ibax developers should connect to platforms they don't allow with ibax wallet removable and better improved security in up-to-date trust wallet and similar wallets

**Comment by doniest** [comment]
aynen katılıyorum

**Comment by ibaxio** [comment, marked as answer]
Thanks for your suggestion.

Please refer to this discussion:  [#629 ](https://github.com/IBAX-io/go-ibax/discussions/214#discussioncomment-2042235)
